code/__DEFINES/admin.dm

```diff
@@ -30,7 +30,7 @@
 #define R_POSSESS		64
 #define R_PERMISSIONS	128
 #define R_STEALTH		256
-#define R_REJUVINATE	512

```

Comment:
You could keep this defined as 2 then your code supports legacy configs (if you keep the parser line, could even have it warn)